docs/advanced-usage.md

```diff
@@ -57,21 +98,21 @@ To restrict access to Advisor by specifying the teams that are authorized to use
 
 1. To limit access, open the OpenAI configuration, mark its **Visibility** as **Private**, and then click the **Ownership** icon:  
    ![OpenAI Connection configuration dialog - Ownership icon](../docs/res/rbac_openai.png)
-2. In the **Assign Owners** dialog, you will see that the logged-in user's teams are automatically assigned ownership. To remove ownership of any team, click the **Red Cross** beside its name; similarly, to assign ownership to any team, select the team from the **Owners** drop-down list, click **Assign** and then click **Submit**:  
+2. In the **Assign Owners** dialog, you will see that the logged-in user's teams are automatically assigned ownership. To remove ownership of any team, click the **Red Cross** beside its name; similarly, to assign ownership to any team, select the team from the **Owners** drop-down list, click **Assign**, and then click **Submit**:  
    ![OpenAI Connection configuration - Assigning owners for the connector](../docs/res/openai_assignownership.png)  
 3. Mark the following playbooks in the '10 - SP - Fortinet Advisor' playbook collection as **Private** and assign their ownership to the same teams that are assigned as owners in the OpenAI connector.
    - Generate WorkPlan For User Input
    - Get Playbook Block Suggestion
    - Converse with LLM  
-     To mark a playbook, for example, the "Converse with LLM" playbook, as 'Private' and restrict its visibility to specific teams, follow these steps:
+     To mark a playbook, for example, the *Converse with LLM* playbook, as 'Private' and restrict its visibility to specific teams, follow these steps:
      1. Open the Converse with LLM playbook in the playbook designer, and in the top bar, mark its **Visibility** as **Private**, and then click the **Ownership** icon.  
         ![Marking playbooks as Private](../docs/res/pb_mark_private.png)  
      2. In the **Assign Owners** dialog, assign the ownership of the playbook to the same teams that are assigned as owners in the OpenAI connector, and then click **Submit**.   
         Once the teams are assigned in both the playbooks and the OpenAI connector, only those teams will have access to the OpenAI connector and will be able to receive responses from Fortinet Advisor.
 
 ## How to update the OpenAI Model for use by Advisor?
 
-You can choose the OpenAI Model that you want Advisor to use by editing the `OpenAI_Model` global variable. By default, Advisor uses the 'gpt-4-1106-preview' model.
+You can choose the OpenAI Model that you want the Advisor to use by editing the `OpenAI_Model` global variable. By default, Advisor uses the 'gpt-4-1106-preview' model.
```

Comment:
We have to remove this statement, Now user can choose OpenAI Model from FortiAI Configuration Wizard instead of Global Variable